### PR TITLE
charm push - fix bzr log if there are merge commit

### PR DESCRIPTION
**Pull request by jrwren**

### CODE REVIEW

**Comment by jrwren** [comment]
fixes https://github.com/juju/charmstore-client/issues/90

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/charmstore-client/167/

**Review by bac — APPROVED** [review]

File: cmd/charm/charmcmd/push.go [hunk 1]
```diff
@@ -437,6 +437,7 @@ func parseBzrLogEntry(entryText string) (bzrLogEntry, error) {
 		val := strings.TrimLeft(kvp[1], " ")
 		switch kvp[0] {
 		case "revno":
+			val = strings.TrimSuffix(val, " [merge]")
```
Comment (cmars) [inline cmd/charm/charmcmd/push.go:440]: This works. Could also just remove the space and anything else following the revno digits (`s/ .*//` for example), not knowing or caring much about the possible things bzr could append to the revno.

**Review by cmars — APPROVED** [review]

**Comment by jrwren** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/charmstore-client-merge

**Pull request merged**